src/metro-config/index.ts

```diff
@@ -25,6 +25,8 @@ const customMappingWatchOptions = {
  * @param metroConfig - configuration of Metro Bundler used in project.
  * @link https://facebook.github.io/metro/docs/configuration
  *
+ * @param enableWatcher - if the custom mapping file should be watched for changes (disable in CI/CD environments)
+ *
```

Comment:
Let's better move it to [EvaConfig](https://github.com/akveo/react-native-ui-kitten/blob/master/src/metro-config/services/eva-config.service.ts#L18), so that we don't specify the large number of params in `create` function:

```ts
export interface EvaConfig {
  evaPackage: EvaMappingPackageName;
  customMappingPath?: string;
  watch?: boolean;
}
```

Should be true by default.

Reply:
Good call, PR has been updated.